content/storing-in-cloud-must-try.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Storing in cloud, must try
+desc: Why i store my docs in cloud
+author: Raymond WP
+github: raywp
+twitter:
+telegram:
+date: 2022-03-10
+cover: https://www.codebase.co.in/wp-content/uploads/2020/02/Choose-Best-Cloud-Storage-Provider-For-Your-Requirement.jpg
+categories:
+  - cloud
+  - storage
+---
+
+## Which cloud?
+
+Any cloud storage as long as it's not the cloud that can cause rain. Goole Drive, OneDrive, DropBox, etc you name it.
+
+## Origin
+
+Once upon a time, when a friend of mine told me "Cloud is new best friend". So i was thinking to give it a try. Back then i thought by separating disk is enough for data backup-ing (Like C: and D:); but in fact, your local disk could be corrupted, your laptop could be stolen, your data could lost. Since at the time i am writing this, I was a final-year student that love my Final Year Project so much, i can't afford to lose my data, **_I need backup_**.
```

Comment:
```suggestion
Once upon a time, a friend of mine told me "Cloud is a new best friend". So I was thinking to give it a try. Back then, I thought separating disks is enough for data backup (Like drive C: and D:); but in fact, your local disk could be corrupted, your laptop could be stolen, your data could be lost. Since at the time I am writing this, I was a final-year student that love my Final Year Project so much, I can't afford to lose my data, **_I need backup_**.
```